openpype/hosts/maya/plugins/load/load_audio.py

```diff
@@ -30,7 +30,7 @@ def load(self, context, name, namespace, data):
             file=context["representation"]["data"]["path"], offset=start_frame
```

Comment:
Interesting approach - I have a feeling that this should actually be `self.filepath_from_context(context)` instead of just taking it from the repsesentation's data directly. I have a feeling that might provide issues since (OR nobody knew it's there?) since I haven't seen that data entry used before.

Does that still work with AYON too?

Reply:
Think it might just old code maybe?
Will let @iLLiCiTiT weigh in on this.

Reply:
It is as @BigRoy wrote. It should use `self.filepath_from_context(context)`. This would work only on the platform where it was published (and does not use root overrides).